docs/DeveloperGuide.md

```diff
@@ -408,11 +408,12 @@ _{Explain here how the data archiving feature will be implemented}_
 
 **Target user profile**:
 
-* financial advisors
+* financial advisors based in Singapore and deals with Singaporean clients
 * has a need of scheduling meetings with a significant number of clients
 * can type fast
 * prefers typing to mouse interactions
 * is reasonably comfortable using CLI apps
+* has English as his first language
```

Comment:
Maybe
> can comfortably read and type in English
will be better, since English dosent really have to be their first language to use our application